### DISCUSSION TITLE
Show fields in Many-to-any (M2A) listing

### DISCUSSION THREAD

**Discussion by nagman** [directus/directus#3717] [Feature Requests]
Currently, there's no way to display nested fields of the selected entries in a M2A field.

![image](https://user-images.githubusercontent.com/16116486/104809560-87604b80-57ee-11eb-9d48-2b4b65804474.png)

I suppose this is in the pipeline.

Here's a related post: https://github.com/directus/directus/issues/3601

**Comment by benhaynes** [comment]
I believe this uses the display template on each of those related collection's settings. I thought it allowed for using nested data, but I would need to confirm.

**Reply by nagman** [reply]
Yes! You're right! I've changed the display settings in one of the collections and it changed the layout in my M2A field
